File: ios/Sources/AppConfig.swift
import Foundation

enum AppConfig {
    static var baseURL: URL {
        if let raw = ProcessInfo.processInfo.environment["CRICKETCLUBAPP_BASE_URL"],
           let url = URL(string: raw.trimmingCharacters(in: .whitespacesAndNewlines)),
           !raw.isEmpty {
            return url
        }
        return URL(string: "https://cricketcanclubs-web.azurewebsites.net")!
    }

    static var healthURL: URL {
        baseURL.appendingPathComponent("api/health")
    }
}

File: android/app/src/main/java/com/cricketclubapp/AppConfig.kt
package com.cricketclubapp

import android.os.Build

object AppConfig {
    val baseUrl: String
        get() {
            val env = System.getenv("CRICKETCLUBAPP_BASE_URL")?.trim().orEmpty()
            if (env.isNotEmpty()) return env.trimEnd('/')
            return "https://cricketcanclubs-web.azurewebsites.net"
        }

    val healthUrl: String
        get() = "$baseUrl/api/health"
}
